Mathlib/Geometry/Manifold/VectorBundle/SmoothSection.lean

```diff
@@ -31,6 +31,185 @@ variable (F : Type*) [NormedAddCommGroup F] [NormedSpace 𝕜 F]
   -- `V` vector bundle
   [∀ x : M, TopologicalSpace (V x)] [FiberBundle F V]
 
+-- Binary and finite sums and scalar products of smooth sections are smooth
+section operations
+
+-- Let V be a vector bundle
+variable [∀ x, AddCommGroup (V x)] [∀ x, Module 𝕜 (V x)] [VectorBundle 𝕜 F V]
+
+variable {I F n V}
+
+variable {f : M → 𝕜} {a : 𝕜} {s t : Π x : M, V x} {u : Set M} {x₀ : M}
+
+lemma contMDiffWithinAt_add_section
```

Comment:
What about naming this one `ContMDiffWithinAt.add_section` , and similarly for the other ones, to make dot notation usable?